Tests/Selenium2TestCaseTest.php

```diff
@@ -559,12 +571,17 @@ public function testThePageCanBeRefreshed()
 
     public function testLinkEventsAreGenerated()
     {
+        $this->markTestIncomplete("Waiting for new phpunit-selenium release");
```

Comment:
If the `alertIsPresent()` method isn't in the PHPUnit phar the `waitUntil()` method always timeouts (it throws unknown method error until timeout). That's linked with the release problem that I tried to explain.